### DISCUSSION TITLE
Provide a manufacturer string for a node when the device config is unknown

### DISCUSSION THREAD

**Discussion by kpine** [zwave-js/zwave-js#7505] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

https://github.com/home-assistant/core/issues/133518

If a device is unknown (no device config available yet), but the manufacturer is a known one, HA is unable to at least provide the friendly manufacturer name (as it's derived from the config) and shows "unknown" instead. While the numerical ID is provided and could be used, it would required either HA build it's own mfg database or query it from the driver. Preferably, a string version could be provided at the node level (or wherever) that HA could use as a fallback.

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

Interview log is present in the linked issue if needed.

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment, marked as answer]
Escalated to an issue: https://github.com/zwave-js/node-zwave-js/issues/7548
